src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -80,10 +80,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List<Lecture> lectures = getLectures();
-    Gson gson = new Gson();
-    response.setContentType("application/json");
-    response.getWriter().println(gson.toJson(lectures));
+    // TODO: Implement getting one lecture.
```

Comment:
even tho this is missing, we're OK to merge b/c nothing uses this... correct?

Reply:
Yes